src/com/spikes2212/gui/DashBoardController.java

```diff
@@ -0,0 +1,69 @@
+package com.spikes2212.gui;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import edu.wpi.first.wpilibj.Sendable;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class DashBoardController {
+	private Map<String, Supplier<String>> stringFields;
+	private Map<String, Supplier<Double>> doubleFields;
+	private Map<String, Supplier<Boolean>> booleanFields;
+	private Map<String, Supplier<Sendable>> sendableFields;
```

Comment:
In truth, now that I read the Sendable interface, you might just want to toss this our and just write Sendable implementations for Suppliers for strings, doubles and booleans